app/scripts/directives/mobileServiceInstanceRow.js

```diff
@@ -87,14 +89,17 @@
         });
 
         if(!bindingsWatch){
-          bindingsWatch = DataService.watch(APIService.getPreferredVersion("servicebindings"), {namespace: $routeParams.project}, function(bindings){
-            row.bindings = _.filter(bindings.by("metadata.name"), function(binding){
-              return _.get(binding.metadata.annotations, "binding.aerogear.org/consumer") === row.mobileClient.metadata.name &&
-                    _.get(binding.metadata.annotations, "binding.aerogear.org/provider") === row.apiObject.metadata.name;
+          bindingsWatch = DataService.watch(APIService.getPreferredVersion('servicebindings'), {namespace: $routeParams.project}, function(bindings) {
```

Comment:
can we move `APIService.getPreferredVersion('servicebindings')` to the top as a variable?